zio-http/src/main/scala/zio/http/Client.scala

```diff
@@ -216,36 +218,19 @@ object Client {
       ZIO.serviceWithZIO[Client](_.socket(url, app, headers))
     }
 
-  val live: ZLayer[ClientConfig with Scope, Throwable, Client] = ZLayer {
-    for {
-      settings       <- ZIO.service[ClientConfig]
-      channelFactory <- channelFactory(settings)
-      eventLoopGroup <- eventLoopGroup(settings)
-      zx             <- HttpRuntime.default[Any]
-    } yield ClientLive(settings, zx, channelFactory, eventLoopGroup)
-  }
-
-  val default = ClientConfig.default >>> live
-
-  private def channelFactory(config: ClientConfig): UIO[ChannelFactory] = {
-    config.channelType match {
-      case ChannelType.NIO    => ChannelFactory.Live.nio
-      case ChannelType.EPOLL  => ChannelFactory.Live.epoll
-      case ChannelType.KQUEUE => ChannelFactory.Live.kQueue
-      case ChannelType.URING  => ChannelFactory.Live.uring
-      case ChannelType.AUTO   => ChannelFactory.Live.auto
+  val live
+    : ZLayer[ClientConfig with Scope with ChannelFactory with EventLoopGroup with NettyRuntime, Throwable, Client] =
```

Comment:
I see that it was like this before as well, but that `Scope` as input does not seem necessary.